### ISSUE TITLE
On some GNU/Linux systems, the first level will fail to start

### ISSUE THREAD

**Issue by z0u** [z0u/cargo#3]
**Cargo:** 0.92

Loading the first level from the menu screen fails for certain distributions of Linux. The crash is in Python:

```
Traceback (most recent call last):
  File "/home/foo/Games/Cargo/cargo-0.92/assets/Scripts/__init__.py", line 25, in <module>
    import Scripts.menu
  File "/home/foo/Games/Cargo/cargo-0.92/assets/Scripts/menu.py", line 20, in <module>
    import webbrowser
  File "/home/foo/Games/Cargo/cargo-0.92/2.68/python/lib/python3.3/webbrowser.py", line 655, in <module>
    cmd = _synthesize(cmdline, -1)
  File "/home/foo/Games/Cargo/cargo-0.92/2.68/python/lib/python3.3/webbrowser.py", line 97, in _synthesize
    controller = copy.copy(controller)
  File "/home/foo/Games/Cargo/cargo-0.92/2.68/python/lib/python3.3/copy.py", line 89, in copy
    rv = reductor(2)
TypeError: attribute of type 'NoneType' is not callable
```

**Comment by z0u** [comment]
This is due to a bug in Python; see https://developer.blender.org/T36963

**Comment by z0u** [comment]
Fixed upstream; fix available in (at least) Cargo 0.98.
